addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRuleUnitTest.java

```diff
@@ -289,6 +290,22 @@ void shouldNotAlertOnReasonableMetaCsp() {
         assertThat(alertsRaised.size(), equalTo(0));
     }
 
+    @Test
+    void shouldNotFailToScanMetaWithMissingAttributes() {
+        // Given
+        HttpMessage msg =
+                createHttpMessageWithReasonableCsp(HttpFieldsNames.CONTENT_SECURITY_POLICY);
+        msg.setResponseBody(
+                "<html><head><meta http-equiv=\""
```

Comment:
Shouldn’t this be missing then?

Reply:
It's the one in the line 303 (It needs to have at least one valid `meta` element).

Reply:
Okay